apps/src/applab/designMode.js

```diff
@@ -859,7 +859,7 @@ designMode.renderToggleRow = function (screenIds) {
   var designToggleRow = document.getElementById('designToggleRow');
   if (designToggleRow) {
     React.render(
-      React.createElement(DesignToggleRow, {
+      React.createElement(PlaySpaceHeader, {
```

Comment:
What?  That's awesome.  I had no idea.

On Mon, Feb 22, 2016 at 4:44 PM, Brent Van Minnen <notifications@github.com>
wrote:

> In apps/src/applab/designMode.js
> <https://github.com/code-dot-org/code-dot-org/pull/6916#discussion_r53720039>
> :
>
> > @@ -859,7 +859,7 @@ designMode.renderToggleRow = function (screenIds) {
> >    var designToggleRow = document.getElementById('designToggleRow');
> >    if (designToggleRow) {
> >      React.render(
> > -      React.createElement(DesignToggleRow, {
> > +      React.createElement(PlaySpaceHeader, {
>
> This comment isn't particular to this PR, and not something you
> necessarily need to address. But I think with our current build system we
> can using JSX in .js files. This means, we could get rid of the
> React.createElement here and do
>
> <PlaySpaceHeader
>   hideToggle: {Applab.hideDesignModeToggle()}
>   ...
> />
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6916/files#r53720039>.
>



-- 
Brad Buchanan
Software Engineer, Code.org
brad@code.org